Description: drop trybuild dependency and tests that use it
 trybuild compiles crokey, fetching upstream crates. Adn it
 does not seem to respect setting .cargo/config.toml, thanks
 to which patching crates.io source to debian cargo registry
 does not work.
Author: Ananthu C V <weepingclown@disroot.org>
Forwarded: not-needed
Last-Update: 2024-06-28
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -66,4 +66 @@ version = "0.2"
 version = "2"
-
-[dev-dependencies.trybuild]
-version = "1.0.55"
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -351,6 +351,6 @@ mod tests {
 
-    #[test]
-    fn ui() {
-        trybuild::TestCases::new().compile_fail("tests/ui/*.rs");
-    }
+    // #[test]
+    // fn ui() {
+    //     trybuild::TestCases::new().compile_fail("tests/ui/*.rs");
+    // }
 }
